src/app/modules/main/communities/view.nim

```diff
@@ -1,18 +1,22 @@
 import NimQml, json, strutils, json_serialization, sequtils
 
 import ./io_interface
-import ../model, ../item
+import ../model, ../item, ../active_section
```

Comment:
@jrainville if you need `model` and `item` initially used in `src/app/modules/main` module, what do you think of moving them to `src/app/modules/shared_models` and updating code accordingly (whenever more than one module is using model/item, it should be placed to `shared_models` and contains an union of properties needed on each sides)?